rooms.js

```diff
@@ -1453,6 +1472,15 @@ class ChatRoom extends BasicRoom {
 		if (message) message += '</div>';
 		return message;
 	}
+	/**
+	 * @param {boolean?} showSecret
+	 * @return {ChatRoom[]}
+	 */
+	getSubRooms(showSecret) {
```

Comment:
Define this as `showSecret = false` and you can type this `boolean` instead of `boolean?`

Reply:
Ah yeah, I know ``boolean?`` is shitty but I got so fed up with typescripting I had a "fuck it Zarel will probably point it out" moment

Reply:
Also, move this to the `Room` class instead of the `ChatRoom` class.